cmd/workspace.go

```diff
@@ -3,28 +3,32 @@ package cmd
 import (
 	"fmt"
 
+	"github.com/exercism/cli/config"
 	"github.com/spf13/cobra"
 )
 
 // workspaceCmd outputs the path to the person's workspace directory.
 var workspaceCmd = &cobra.Command{
 	Use:     "workspace",
 	Aliases: []string{"w"},
-	Short:   "Output the path to your workspace.",
-	Long: `Output the path to your workspace.
+	Short:   "Print out the path to your Exercism workspace.",
+	Long: `Print out the path to your Exercism workspace.
 
-This command can be combined with shell commands to take you there.
+This command can be used for scripting, or it can be combined with shell
+commands to take you to your workspace.
 
-For example, on Linux or MacOS you can run:
+For example you can run:
 
     cd $(exercism workspace)
 
-On Windows, the equivalent command is:
-
-    TODO ask @exercism/windows for help
+On Windows, this command works in Powershell, however you would need to
```

Comment:
This reference to `this command` may be confusing as to whether it is referring to the workspace command or the `cd ...` command above. Maybe `On Windows, this will work only with Powershell, however ...`